apps/hostd/dialogs/AlertsDialog.tsx

```diff
@@ -19,7 +20,15 @@ type Props = {
 }
 
 export function AlertsDialog({ open, onOpenChange }: Props) {
-  const alerts = useAlerts()
+  const alerts = useAlerts({
```

Comment:
This should always use the default too imo

Reply:
oop, missed that one, ty.